packages/core/src/utils/client.ts

```diff
@@ -0,0 +1,14 @@
+import { SlashauthResponse } from '../client';
+
+export const transformResponse =
+  <I, O>(
```

Comment:
I presume `I` and `O` stand for `input` and `output` respectively? If not then let's rename these to use `T` and `U` as these are more common for generics

Reply:
updated